src/Facebook/PersistentData/FacebookMemoryPersistentDataHandler.php

```diff
@@ -29,28 +29,26 @@
  */
 class FacebookMemoryPersistentDataHandler implements PersistentDataInterface
 {
+    /**
+     * @var array The session data to keep in memory.
+     */
+    protected $sessionData = [];
 
-  /**
-   * @var array The session data to keep in memory.
-   */
-  protected $sessionData = [];
-
-  /**
-   * @inheritdoc
-   */
-  public function get($key)
-  {
-    return isset($this->sessionData[$key])
-      ? $this->sessionData[$key]
-      : null;
-  }
-
-  /**
-   * @inheritdoc
-   */
-  public function set($key, $value)
-  {
-    $this->sessionData[$key] = $value;
-  }
+    /**
+     * @inheritdoc
+     */
+    public function get($key)
+    {
+        return isset($this->sessionData[$key])
+            ? $this->sessionData[$key]
+            : null;
```

Comment:
It's a close call there. That line is 82 characters long on a single line. The linter does not object, so I refactored to one line.

![screenshot](https://dl.dropboxusercontent.com/u/73968/photos/Screen%20Shot%202015-02-10%20at%2011.06.04%20AM.png)